### DISCUSSION TITLE
High Data Usage

### DISCUSSION THREAD

**Discussion by anarkia47** [netalertx/NetAlertX#573] [Q&A]
I've installed  Pi-alert on a Raspberry Pi, and have noticed it consumed over 15GB of data in the last 24hrs.

At this rate, it's using over 1GB per hour.  Is this considered normal  behavior ?  

![pi-alert-3](https://github.com/jokob-sk/Pi.Alert/assets/18148653/0eac9618-416a-4a11-94e0-c7cafbb2f22a)
![pi-alert-2](https://github.com/jokob-sk/Pi.Alert/assets/18148653/ea7d62e0-d8ce-4ff6-ab96-8c2d422fa16f)
![Screenshot from 2024-02-19 17-11-31](https://github.com/jokob-sk/Pi.Alert/assets/18148653/9c865fbc-8493-48bc-8f45-cf38977e1843)

**Comment by jokob-sk** [comment]
Hi @anarkia47 ,

You can try to decrease the frequency of the scans to see if that influences the bandwidth usage. 

I checked my server and it uses 24Gb /24h but it's a NAS, so that's expected. 

Could you please double-check if that's the PiAlert container causing this much data usage? Also, can you please check that it is not restarting (in `pialert.log`), which might cause the assets to re-download?

Thanks in advance,
j
